<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>278</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>278</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BMP to STR converter</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Locate the input image file</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_InputFileLocation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInputImageFile">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonStartConversion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Start conversion!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
